diff --git a/node_modules/native-base/lib/typescript/theme/index.d.ts b/node_modules/native-base/lib/typescript/theme/index.d.ts
index 837bb07..d381ca3 100644
--- a/node_modules/native-base/lib/typescript/theme/index.d.ts
+++ b/node_modules/native-base/lib/typescript/theme/index.d.ts
@@ -2943,7 +2943,16 @@ declare const theme: {
         success: {};
         warning: {};
         muted: {};
-        primary: {};
+        primary: {50: string;
+            100: string;
+            200: string;
+            300: string;
+            400: string;
+            500: string;
+            600: string;
+            700: string;
+            800: string;
+            900: string;};
         info: {};
         secondary: {};
         light: {};
